setup.py

```diff
@@ -11,16 +11,14 @@
     author_email='aroy@alum.mit.edu',
     url='https://github.com/audreyr/cookiecutter-pypackage',
     keywords=['cookiecutter', 'template', 'package', ],
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
+    python_requires='>=3.5, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
```

Comment:
I'd recommend removing that, as I'd also recommend new packages using the cookie cutter should also only support current Python versions, ie. 3.5+.

```suggestion
    python_requires='>=3.5',
```

Reply:
Added in latest commit